research/fanout-cross-series-compression/01-transposed-for.prompt.md

```
@@ -0,0 +1,107 @@
+  3. Store the transposed encoded buffer plus metadata needed for decoding
+  4. Implement the full StorageBackend interface so queries still work
+
+- You MUST also test a **hybrid approach**: 
+  - Use ALP per-series encoding for the *within-series temporal* axis (existing codec)
+  - Then apply a cross-series FoR pass on the ALP-encoded integers (before bit-packing)
+  - Compare: pure transposed vs hybrid vs current independent ALP
+
+- You MUST add the new backend(s) to `packages/o11ytsdb/bench/engine.bench.ts` alongside existing backends
+
+- You MUST run the engine benchmark:
+  ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)